packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -2329,7 +2347,7 @@ export default class Meeting extends StatelessWebexPlugin {
         remoteQualityLevel: this.mediaProperties.remoteQualityLevel
       })
         .then((peerConnection) => {
-          LoggerProxy.logger.log(`Meeting:index#addMedia --> PeerConnection Received from attachMedia ${peerConnection}`);
+          LoggerProxy.logger.log(`${LOG_HEADER} PeerConnection Received from attachMedia ${peerConnection}`);
```

Comment:
```suggestion
          LoggerProxy.logger.info(`${LOG_HEADER} PeerConnection Received from attachMedia ${peerConnection}`);
```
Probably can drop the level of this, most consumers won't need this info.